data_structures/binary_trees/threaded_binary_trees.c

```diff
@@ -1,15 +1,13 @@
 /**
  * @file
- * \brief Threaded Binary Tree is a binary tree variant in which all left child
- * pointers that are NULL (in Linked list representation) point to its
- * in-order predecessor, and all right child pointers that are NULL
```

Comment:
accidentally deleted the details 😨

Reply:
I thought that was unnecessary, added them now.